public/pages/workflow_detail/workflow_inputs/processor_inputs/ml_processor_inputs.tsx

```diff
@@ -290,6 +300,23 @@ export function MLProcessorInputs(props: MLProcessorInputsProps) {
       {!isEmpty(getIn(values, modelFieldPath)?.id) && (
         <>
           <EuiSpacer size="s" />
+          {props.context === PROCESSOR_CONTEXT.SEARCH_REQUEST && (
+            <>
+              <EuiText
+                size="m"
+                style={{ marginTop: '4px' }}
+              >{`Override query (Optional)`}</EuiText>
+              <EuiSpacer size="s" />
+              <EuiSmallButton
+                style={{ width: '100px' }}
+                fill={false}
+                onClick={() => setIsQueryModalOpen(true)}
+              >
+                Override
+              </EuiSmallButton>
```

Comment:
@Tyler, when adding buttons can we also add data-testid. That helps in future when writing tests.

Reply:
sure will add